app/src/main/java/fr/neamar/kiss/MainActivity.java

```diff
@@ -297,6 +310,16 @@ protected void onResume() {
             hideKeyboard();
         }
 
+        if (prefs.getBoolean("enable-spellcheck", true)) {
+            searchEditText.setInputType(spellcheckEnabledType);
+        }
+        else {
+            searchEditText.setInputType(spellcheckDisabledType);
```

Comment:
hum, thinking on this: we don't really need a `else`, since the default inputType (from the xml layout) is already correct.

The only case this could be useful is right after changing theme, and there's no need to set this on all other case.

Best thing i can think of is reusing the `require-layout-update` flag when switching the toggle.

That way, we can `setInputType()` on onCreate() once, and only if it's required \o/